proto/cosmwasm/wasm/v1/tx.proto

```diff
@@ -388,4 +391,20 @@ message MsgStoreAndInstantiateContractResponse {
   string address = 1;
   // Data contains bytes to returned from the contract
   bytes data = 2;
-}
\ No newline at end of file
+}
+
+// MsgPruneWasmCodes is the MsgPruneWasmCodes request type.
+message MsgPruneWasmCodes {
+  option (amino.name) = "wasm/MsgPruneWasmCodes";
+  option (cosmos.msg.v1.signer) = "authority";
+
+  // Authority is the address of the governance account.
+  string authority = 1 [ (cosmos_proto.scalar) = "cosmos.AddressString" ];
+  // LatestCodeID references the upper limit until unpinned
+  // code ids will be pruned
```

Comment:
Could you be explicit here if there upper limit is included or not? "latest code ID" sounds lit it is included but the implementation seems to be <.

I think <= is a bit simpler to reason about.